src/refract/VisitorUtils.cc

```diff
@@ -0,0 +1,44 @@
+//
+//  refract/VisitorUtils.cc
+//  librefract
+//
+//  Created by Vilibald Wanča on 09/11/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+#include "Element.h"
+#include "Visitors.h"
+#include "VisitorUtils.h"
+
+namespace refract
+{
+
```

Comment:
You haven't moved the 3 existing functions from the `.h` file to this `.cc` file.